src/GafferSceneUI/SceneView.cpp

```diff
@@ -1072,8 +1094,19 @@ class SceneView::Camera : public boost::signals::trackable
 			SetFilterPtr lightFilter = new SetFilter;
 			lightFilter->setExpressionPlug()->setValue( "__lights" );
 
+			m_distantApertureAttributeQuery->scenePlug()->setInput( view->inPlug<ScenePlug>() );
+			m_distantApertureAttributeQuery->setup( new Gaffer::FloatPlug() );
```

Comment:
This is a memory leak, because `setup()` doesn't take ownership of the argument, and we don't free it.

Reply:
Fixed in place.